src/renderer/services/utils/crypto-utils.ts

```diff
@@ -11,17 +12,59 @@ export function toHexString(data: string | Buffer | BN): string {
 }
 
 /**
- * Provides methods to convert amount defined in eth to wei / gwei.
+ * Provides methods to convert amounts from currencies defines as @{Currency}.
  */
 export const EthConverter = {
```

Comment:
@mpetrunic We still need conversion util because ethers util methods only cover _eth -> wei_ conversion. Maybe I can fully extract this utility and publish it over npm as lib?